docker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '2'
+services:
+    opencats:
+      image: prooph/nginx:www
+      ports:
+          - "80:80"
+          - "443:443"
+      volumes_from:
+        - dataphp
+
+    php:
+      build: ./php/
+      volumes_from:
+        - dataphp
+
+    dataphp:
+      image: debian:jessie
+      volumes:
+        - ../:/var/www/public
```

Comment:
this data image shall be small, better to base it probably on busy box. Not necessary to be running. so proposed:
```yml
dataphp:
    image: busybox
    volumes:
        - ..:/var/www/public
    command: "true"
```